### PR TITLE
8340799: Add border inside instruction frame in PassFailJFrame

### PR DESCRIPTION
**Pull request by aivanov-jdk**
This changeset unifies borders used inside the `PassFailJFrame` framework and adds missing borders.

I started it in  for [JDK-8340306](https://bugs.openjdk.org/browse/JDK-8340306). That changeset handled only the instructions.

This changeset add a border around the entire contents of the instruction frame, adds a border to the log area as well as to the failure reason dialog. The result looks more pleasing and consistent.

I chose the gap of 4 because it plays nicely with fractional scales: 4 × 1.25 = 5 — no fractional pixels.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8340799](https://bugs.openjdk.org/browse/JDK-8340799): Add border inside instruction frame in PassFailJFrame (**Enhancement** - P4)

### Reviewers
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**)
 * [Damon Nguyen](https://openjdk.org/census#dnguyen) (@DamonGuy - Committer)
 * @abdelhak-zaaim (no known openjdk.org user name / role)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/21327/head:pull/21327` \
`$ git checkout pull/21327`

Update a local copy of the PR: \
`$ git checkout pull/21327` \
`$ git pull https://git.openjdk.org/jdk.git pull/21327/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 21327`

View PR using the GUI difftool: \
`$ git pr show -t 21327`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/21327.diff">https://git.openjdk.org/jdk/pull/21327.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/21327#issuecomment-2391679017)

### CODE REVIEW

File: test/jdk/java/awt/regtesthelpers/PassFailJFrame.java [hunk 1]
```diff
@@ -664,7 +668,7 @@ private static JComponent createInstructionUIPanel(String instructions,
         text.setEditable(false);
 
         JPanel textPanel = new JPanel(new BorderLayout());
-        textPanel.setBorder(createEmptyBorder(4, 0, 0, 0));
+        textPanel.setBorder(createEmptyBorder(GAP, 0, GAP, 0));
         textPanel.add(new JScrollPane(text), BorderLayout.CENTER);
```
Comment (aivanov-jdk) [inline test/jdk/java/awt/regtesthelpers/PassFailJFrame.java:672]: I accidentally committed these three lines in  for [JDK-8340785](https://bugs.openjdk.org/browse/JDK-8340785) and I didn't notice it.

This three-line snippet was completely unrelated to updating description of the `PassFailJFrame` class and samples.

**Comment by aivanov-jdk** [comment]
A couple of screenshots for comparison.

**Before:**  
Screenshot of PrintLatinCJKTest where no borders are added to `PassFailJFrame` instruction UI

**After:**  
Screenshot of PrintLatinCJKTest with added borders in `PassFailJFrame` instruction UI

**Review by prrace — APPROVED** [review]

**Review by DamonGuy — APPROVED** [review]
Changes look good! The "after" screenshot does look better with a gap.

**Review by abdelhak-zaaim — APPROVED** [review]

**Pull request closed without merging**